casper/src/main/scala/coop/rchain/casper/util/ProtoUtil.scala

```diff
@@ -432,34 +433,39 @@ object ProtoUtil {
     )
   }
 
-  def sourceDeploy(source: String, timestamp: Long, phlos: Long): DeployData =
-    DeployData(user = ByteString.EMPTY, timestamp = timestamp, term = source, phloLimit = phlos)
+  def sourceDeploy(source: String, timestamp: Long, phlos: PhloLimit): DeployData =
+    DeployData(
+      user = ByteString.EMPTY,
+      timestamp = timestamp,
+      term = source,
+      phloLimit = Some(phlos)
+    )
 
   def compiledSourceDeploy(
       source: CompiledRholangSource,
       timestamp: Long,
-      phloLimit: Long
+      phloLimit: PhloLimit
```

Comment:
for me is confusing to have three types Cost, PhloLimit and PhloPrice representing the same thing. Why cannot be use Cost everywhere?

Reply:
PhloLimit and PhloPrice are different things. The `Cost` from the `test` directory will soon be gone but the `Cost` from the `rholang` module will stay.